assopy/forms.py

```diff
@@ -182,7 +182,7 @@ def clean_captcha_answer(self):
 
     def clean_email(self):
         email = self.cleaned_data['email']
-        if auth.models.User.objects.filter(email__iexact=email).count() > 0:
+        if auth.models.AssopyUser.objects.filter(email__iexact=email).count() > 0:
```

Comment:
that's not going to work, does it?

Reply:
Thanks for catching this, seems I got too eager renaming the model :)